src/modules/mga/mga.service.ts

```
@@ -281,44 +241,147 @@ export class MgaService {
         return validatedData;
     }
 
-    // Insertar datos en la base de datos
+    // Insertar datos en la base de datos en lotes
     private async createMgaRecords(mgaRecords: CreateMgaDto[]): Promise<any[]> {
         try {
-            const { data, error } = await this.supabaseService.clientAdmin
+            const batchSize = 500; // Inserción de 500 registros por lote
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)